util/bloom_test.cc

```diff
@@ -458,6 +473,172 @@ TEST_F(FullBloomTest, Schema) {
   ResetPolicy();
 }
 
+struct RawFilterTester {
+  std::array<char, 3000> data_;
+  Slice cur_slice_;
+  char* metadata_ptr_;
+
+  RawFilterTester() : metadata_ptr_(&data_.back() - 4) {}
+
+  Slice& ResetNoFill(uint32_t len_without_metadata, uint32_t num_lines,
+                     uint32_t num_probes) {
+    metadata_ptr_[0] = static_cast<char>(num_probes);
+    EncodeFixed32(metadata_ptr_ + 1, num_lines);
+    cur_slice_ = Slice(metadata_ptr_ - len_without_metadata,
+                       len_without_metadata + /*metadata*/ 5);
+    return cur_slice_;
+  }
+
+  Slice& Reset(uint32_t len_without_metadata, uint32_t num_lines,
+               uint32_t num_probes, bool fill_ones) {
+    data_.fill(fill_ones ? 0xff : 0);
+    return ResetNoFill(len_without_metadata, num_lines, num_probes);
+  }
+
+  Slice& ResetWeirdFill(uint32_t len_without_metadata, uint32_t num_lines,
+                        uint32_t num_probes) {
+    for (uint32_t i = 0; i < data_.size(); ++i) {
+      data_[i] = static_cast<char>(0x7b7b >> (i % 7));
+    }
+    return ResetNoFill(len_without_metadata, num_lines, num_probes);
+  }
+};
+
+TEST_F(FullBloomTest, RawSchema) {
+  RawFilterTester cft;
+  // Two probes, about 3/4 bits set: ~50% "FP" rate
+  // One 256-byte cache line.
+  OpenRaw(cft.ResetWeirdFill(256, 1, 2));
+  ASSERT_EQ(PackedMatches(), 18446744073675927543ULL);
```

Comment:
Not the big deal but I believe it is `ASSERT_EQ(<expected_value>, <actual_value>)`, as the message printed out when assertion fails. No need to fix everywhere, just in case we'll need to add more tests in the future.

Reply:
Message doesn't assume that for me, but I'll go for consistency.

Expected equality of these values:
  PackedMatches()
    Which is: 18446744073675927543
  18446744073675927544ULL
    Which is: 18446744073675927544